fda-model/metrics/Results.cs

```diff
@@ -20,33 +23,34 @@ public class Results: IContainResults
         }
         public Results(){
             _aepThreshold = 0.0;
-            _meanAEP = 0.0;
-            _aepCount = 0;
-            _meanEADs = new Dictionary<string, double>();
-            _eadCounts = new Dictionary<string, Int64>();
+            _aep = new Histogram(null, AEP_HISTOGRAM_BINWIDTH);
```

Comment:
if null doesnt work on the Idata element, just set the whole AEP histogram to null here.